clients/drcachesim/common/trace_entry.h

```diff
@@ -612,6 +612,13 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_CONTEXT_SWITCH_END,
 
+    /**
+     * The marker is used to indicate the current vector length in bytes for
```

Comment:
nit: All other marker docs explicitly say "the marker value contains"; that would make it crystal clear that the length is in fact in the value (and not indicated in some other way...I agree I don't know how but good to match the others and be very clear).

Reply:
Done.